Packages/Ink/Editor/Core/InkImporter.cs

```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+using UnityEditor.AssetImporters;
+using System.IO;
+using Ink.UnityIntegration;
+using UnityEditor;
+using Ink;
+
+/// <summary>
+/// Automatically compiles .ink assets each time they are imported, and creates an InkFile asset.
+/// </summary>
+[ScriptedImporter(1, "ink")]
+public class InkImporter : ScriptedImporter
```

Comment:
A complimentary `InkImporterEditor : ScriptedImporterEditor` could be created for better UX.